kinyarwanda-net/python-backend
# Allowing the app to use nodejs

Description:  Allowing the app to use nodejs

Delivers (Node-server)

https://trello.com/c/cFRMRkn3/13-creating-a-nodejs-server

#### What does this PR do?

This PR set the application to use nodejs as the product is moving from using python to using JavaScript. 

#### Description of Task to be completed?
N/A
#### How should this be manually tested?
Before cloning make sure that you have 
`node v11+` installed & 
`npm v6+` installed 

You can download these from the official node website: $ https://nodejs.org 

After cloning the repo: 
- Run `npm install` this cmd will install all dependencies. 
- Run `git checkout KN-node-server-ft` 
- Run `npm run dev` to run the code on dev env. 
- Run `npm run test` to run the tests on test env

#### Any background context you want to provide?

Kinyarwanda.net was initially built in PHP, now the team is moving on to building the product using nodejs. 

#### What are the relevant Trellor stories?

https://trello.com/c/cFRMRkn3/13-creating-a-nodejs-server

#### Screenshots (if appropriate)
`The server is running` 

![image](https://user-images.githubusercontent.com/44478388/74325279-fc323680-4d90-11ea-8c01-6d4f07934145.png)

`The tests are passing `

![image](https://user-images.githubusercontent.com/44478388/74325289-02281780-4d91-11ea-8cad-aac74aa49cd5.png)

#### Questions:
NONE
